test_invalid.go

```diff
@@ -0,0 +1,170 @@
+package main
```

Comment:
the `test` in the file path let's me think that this is a test file. Maybe stick with `invalid.go`